pages/admin/AdminHandler.php

```diff
@@ -552,4 +569,154 @@ protected function getJobsTableState(int $page = 1): array
             'isLoadingItems' => false,
         ];
     }
+
+    /**
+     * List the queue jobs failied to execute
+     *
+     * @param array $args
+     * @param PKPRequest $request
+     */
+    public function failedJobs($args, $request)
+    {
+        $this->setupTemplate($request, true);
+
+        $templateMgr = TemplateManager::getManager($request);
+
+        $page = (int) ($request->getUserVar('page') ?? 1);
+
+        $breadcrumbs = $templateMgr->getTemplateVars('breadcrumbs');
+        $breadcrumbs[] = [
+            'id' => 'failedJobs',
+            'name' => __('navigation.tools.jobs.failed'),
+        ];
+
+        $templateMgr->setState($this->getFailedJobsTableState($request, $page));
+
+        $templateMgr->assign([
+            'pageComponent' => 'FailedJobsPage',
+            'breadcrumbs' => $breadcrumbs,
+            'pageTitle' => 'navigation.tools.jobs.failed',
+        ]);
+
+        $templateMgr->display('admin/failedJobs.tpl');
+    }
+
+    /**
+     * Build the state data for the queued jobs table
+     */
+    protected function getFailedJobsTableState(PKPRequest $request, int $page = 1): array
+    {
+        $total = Repo::failedJob()->total();
+
+        $failedJobs = Repo::failedJob()
+            ->setOutputFormat(Repo::failedJob()::OUTPUT_HTTP)
+            ->perPage(self::JOBS_PER_PAGE)
+            ->setPage($page)
+            ->showJobs();
+
+        return [
+            'label' => __('navigation.tools.jobs.failed.view'),
+            'description' => __('admin.jobs.failed.totalCount', ['total' => $total]),
+            'columns' => [
+                [
+                    'name' => 'id',
+                    'label' => __('admin.jobs.list.id'),
+                    'value' => 'id',
+                ],
+                [
+                    'name' => 'title',
+                    'label' => __('admin.jobs.list.displayName'),
+                    'value' => 'displayName',
+                ],
+                [
+                    'name' => 'queue',
+                    'label' => __('admin.jobs.list.queue'),
+                    'value' => 'queue',
+                ],
+                [
+                    'name' => 'connection',
+                    'label' => __('admin.jobs.list.connection'),
+                    'value' => 'connection',
+                ],
+                [
+                    'name' => 'failed_at',
+                    'label' => __('admin.jobs.list.failedAt'),
+                    'value' => 'failed_at',
+                ],
+                [
+                    'name' => 'actions',
+                    'label' => __('admin.jobs.list.actions'),
+                    'value' => 'action',
+                ],
+            ],
+            'rows' => collect($failedJobs->all())->map(fn($failedJob) => array_merge($failedJob, [
+                'detailsPath' => $request->getDispatcher()->url($request, Application::ROUTE_PAGE, 'index', 'admin', 'failedJobDetails', $failedJob['id'])
```

Comment:
Our standard approach is to include a `_href` attribute which is a URL to the resource in the API. I'd be tempted to put this into your `HTTP****Resource.php` files. That is the equivalent of maps in our schema-based entities.

Reply:
done .